models/users.js

```diff
@@ -54,11 +54,21 @@ const addOrUpdate = async (userData, userId = null) => {
  * @return {Promise<userModel|Array>}
  */
 const fetchUsers = async (query) => {
+  const allowedUserTypes = ['new', 'blocked', 'member']
```

Comment:
Please update the `swaggerDocs` and generate the API Schema accordingly by running the command
`npm run generate-api-schema`

Reply:
Updated swaggerDefinition.js file with additional field "userType".